src/ert/config/gen_kw_config.py

```diff
@@ -313,22 +292,7 @@ def read_from_runpath(
         real_nr: int,
         iteration: int,
     ) -> xr.Dataset:
-        keys = [e.name for e in self.transform_functions]
-        if not self.forward_init_file:
-            raise ValueError("loading gen_kw values requires forward_init_file")
-
-        parameter_value = self._values_from_file(
-            substitute_runpath_name(self.forward_init_file, real_nr, iteration),
-            keys,
-        )
-
-        return xr.Dataset(
-            {
-                "values": ("names", parameter_value),
-                "transformed_values": ("names", self.transform(parameter_value)),
-                "names": keys,
-            }
-        )
+        raise NotImplementedError()
```

Comment:
Why not remove the function altogether?

Reply:
It is a mandatory API (`@abstractmethod`) in `ParameterConfig`.